### DISCUSSION TITLE
Display thumbnails in element selection fields (Entries, etc.)

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9168] [Ideas]
> Created by: Timothy Kelty (timkelty@gmail.com) on 2015/04/02 12:33:05 +0000
> Votes at time of UserVoice import: 6

-----------------------------------

Related to: #884

When you make a selection in an Entries field, or any related element field, the selection is shown by the status dot and title.

It would be great to be able to customize what fields are shown here. Namely, I want to be able to show a thumbnail from an Asset field, alongside the title.

An idea to configure this could be a setting on entry type where you select which fields will show in selections.

**Comment by angrybrad** [comment]
> Posted by Jacques Letesson (jacques@basedigital.io) on 2016/02/29 17:19:58 +0000
-----------------------------------

Couldn&#39;t agree more.

**Comment by d-simon** [comment]
I've been requested to display thumbnails in the select modal, but I'm way in over my head to implement a custom relation field with it's dedicated select modal.
Is this somehow achievable at this point in time?

**Comment by brandonkelly** [comment]
@d-simon if you just want to see thumbnails in the select modal, go to the Entires index page, click the cog icon below the list of sections, choose the section you are relating content from, and tick the checkbox next to your relation’s field name. That will cause the relation to show up as an extra column in both the Entries index, and within entry selection modals.

**Comment by phillubliner** [comment]
This seems plausible as a custom plugin, but would be ideal to have as a native feature/option for entries fields. The title and status dot aren't always so helpful to clients when certain kinds of entry titles may be redundant or non-descriptive … thumbnails are then very useful for quick reference and sorting at the field view level when they're already selected (not just within entry selection modal).
